Makefile

```diff
@@ -51,13 +54,26 @@ test-coverage:
 	poetry run coverage report --show-missing
 	poetry run coverage xml
 
+generate-man:
+	@export QPC_VAR_CURRENT_YEAR=$(CURRENT_YEAR) \
```

Comment:
This works, why `CURRENT_YEAR` is not prefixed with `QPC_VAR_`? That would be simpler and cleaner...